noxfile.py

```diff
@@ -157,6 +157,38 @@ def install_ci(session, group):
             )
 
 
+@nox.session
+def configure_coverage(session) -> None:
+    """Write a coverage config file, adding extra patterns to omit."""
+    import tomlkit
+
+    groups_str = session.posargs[0]  # first positional argument
+
+    print(groups_str)  # for debugging
```

Comment:
Nit: Should this still be there?

Reply:
Yes, we'll need this when we change the group names; this will soon happen as the CI architecture is far from optimal right now.

It's not super intuitive to pass these inputs from the other job around because it's just a long string from the CLI argument. So the easiest is to see it. :)